package/yast2-registration.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+
+Mon May  8 13:27:09 UTC 2017 - knut.anderssen@suse.com
+
+- Don't crash if the regurl provided by linuxrc is invalid using
```

Comment:
Better wording: ... if ... is invalid, use the one...

Again, avoid linuxrc term.